frequency-chain/frequency-xyz
# Chore/update dev portal page to latest design

# Description

The design for the image and text for the main Dev Portal page has changed. Update the page to the current design.

- Closes #67 

## Screenshot(s)
<img width="920" alt="image" src="https://github.com/user-attachments/assets/7c35a007-a59e-487c-a512-545f7b88e833">


## Type of change

Please delete options that are not relevant.

- [x] Other - page did not match design

# How to Test?

1. Pull and run locally
2. Go to "Developer Portal"
3. See that the top half of the page matches the current design (see screenshot)

# Checklist:
